### PR TITLE
Add setup instructions and clarify Prerequisites

### LINKED ISSUE #200: Setup page is empty and links to template
http://swcarpentry.github.io/sql-novice-survey/setup/

### PR DESCRIPTION
**Pull request by gcapes**
Fix #200.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -3,4 +3,11 @@ layout: page
 title: Setup
 permalink: /setup/
 ---
-See the [workshop template page]({{ site.workshop_site }}) for setup instructions.
+# Software
+For this course you will need the UNIX shell, plus [SQLite3](http://www.sqlite.org/) or
+[DB Browser for SQLite](http://sqlitebrowser.org/).
+
+If you are running Linux, you will already have SQLite3 installed.
```
Comment (remram44) [inline setup.md]: This is not usually the case (in particular it's not on Ubuntu)

Comment (henrykironde) [reply setup.md]: Or may be change it to.

If you are running Linux, you may already have SQLite3 installed, if not run 
```
sudo apt-get update
sudo apt-get install sqlite3 libsqlite3-dev
```
and run `sqlite3 --version` to confirm the installation

**Comment by henrykironde** [comment]
Other than the changes requested by @remram44, this PR LGTM.

File: setup.md (hunk 1 above)
Comment (remram44) [reply setup.md]: `apt install sqlite3` should be enough. I don't know if we should add instructions for other distributions...

Comment (henrykironde) [reply setup.md]: which distributions?

**Comment by henrykironde** [comment]
Thanks @gcapes and @remram44.

**Pull request merged**